internal/distro/rhel7/distro.go

```diff
@@ -61,7 +62,7 @@ var distroMap = map[string]distribution{
 		releaseVersion:     "7",
 		modulePlatformID:   "platform:el7",
 		vendor:             "redhat",
-		runner:             "org.osbuild.rhel7",
+		runner:             &runner.RHEL{Major: uint64(7), Minor: uint64(9)},
```

Comment:
nitpick: technically, there is no `rhel79` runner (see https://github.com/osbuild/osbuild/tree/main/runners). However this will work with osbuild v69+ (https://github.com/osbuild/osbuild/pull/996). The SPEC file specifies Requires for osbuild-70, so this should be fine.

Reply:
With the new runners specified in composer we need to specify a minor version and since we're _technically_ building a 7.9 (specified in the `distro.osVersion`), and given the fallback mechanisms in osbuild now, I think this is more correct than 7.0.

Reply:
Oh, I didn't mean that we should use `rhel70` runner, but `rhel7` would be ideal 😅